File: 5-kyu/convert-a-hex-string-to-rgb/python/solution.py
def hex_string_to_RGB(hex_string): 
    hex_string = hex_string[1:].upper()
    return {'r': int(hex_string[:2], 16), 'g': int(hex_string[2:4], 16), 'b': int(hex_string[4:], 16)}

File: 5-kyu/convert-a-hex-string-to-rgb/javascript/solution.js
const hexStringToRGB = hexString => {
  hexString = hexString.slice(1).toUpperCase();
  return { r: parseInt(hexString.slice(0, 2), 16),
          g: parseInt(hexString.slice(2, 4), 16),
          b: parseInt(hexString.slice(4), 16) };
}